src/main/resources/db/scripts.sql

```diff
@@ -42,14 +42,31 @@ CREATE TABLE IF NOT EXISTS app_user
 INSERT INTO app_user(first_name, last_name, username, password, email)
 VALUES ('Mike', 'Traktor', 'MikeUser', 'pass123', 'mike@email.com');
 INSERT INTO app_user(first_name, last_name, username, password, email)
-VALUES ('Alex', 'Committer','AlexUser', 'pass123', 'alex@email.com');
+VALUES ('Alex', 'Committer', 'AlexUser', 'pass123', 'alex@email.com');
 INSERT INTO app_user(first_name, last_name, username, password, email)
-VALUES ('Andreea', 'TalkToUS','AndreeaUser', 'pass123', 'andreea@email.com');
+VALUES ('Andreea', 'TalkToUS', 'AndreeaUser', 'pass123', 'andreea@email.com');
 INSERT INTO app_user(fk_role, first_name, last_name, username, password, email)
 VALUES (2, 'John', 'Doe', 'JohnUser', 'pass123', 'john@email.com');
 INSERT INTO app_user(fk_role, first_name, last_name, username, password, email)
-VALUES (3, 'Jane',  'Dow','JaneUser', 'pass123', 'jane@email.com');
+VALUES (3, 'Jane', 'Dow', 'JaneUser', 'pass123', 'jane@email.com');
 INSERT INTO app_user(first_name, last_name, username, password, email)
 VALUES ('Elena', 'Tintiri', 'ElenaUser', 'pass123', 'elena@email.com');
 INSERT INTO app_user(fk_role, first_name, last_name, username, password, email)
 VALUES (2, 'David', 'Beckham', 'DavUser', 'pass123', 'david@email.com');
+
+CREATE TABLE IF NOT EXISTS address
+(
+    id                 BIGINT      NOT NULL DEFAULT NEXTVAL('seq_address'),
+    country            VARCHAR(45) NOT NULL,
+    city               VARCHAR(45) NOT NULL,
+    door_number        VARCHAR(10) NOT NULL,
+    additional_details VARCHAR(65) NULL,
+    PRIMARY KEY (id)
```

Comment:
CONSTRAITN address_pk PRIMARY KEY (pk). Please follow the same design - look at the app_user table